Source/Urho3D/Urho2D/AnimatedSprite2D.cpp

```diff
@@ -306,6 +306,10 @@ void AnimatedSprite2D::UpdateSpineAnimation(float timeStep)
     worldBoundingBoxDirty_ = true;
 }
 
+typedef enum {
```

Comment:
When is this enum used?

Reply:
This enum was part of the spine runtime, in older versions it was inside **RegionAttachment.h**
It's no longer there so, all the references to its values (e.g. **slotVertices[SP_VERTEX_X1]**) won't compile if we don't define it manually.

What happened is that it got moved into the source file (**RegionAttachment.c**) so it's no longer accessible from the header, and its values now have different names.

```c++
typedef enum { 
BLX = 0, BLY, ULX, ULY, URX, URY, BRX, BRY 
} spVertexIndex;
```

Alternatively we could use these new names to match theirs, in that case the code should look like this:

```c++
vertices[0].position_ = worldTransform * Vector3(slotVertices[BLX], slotVertices[BLY]);
vertices[1].position_ = worldTransform * Vector3(slotVertices[ULX], slotVertices[ULY]);
vertices[2].position_ = worldTransform * Vector3(slotVertices[URX], slotVertices[URY]);
vertices[3].position_ = worldTransform * Vector3(slotVertices[BRX], slotVertices[BRY]);

vertices[0].color_ = color;
vertices[1].color_ = color;
vertices[2].color_ = color;
vertices[3].color_ = color;

vertices[0].uv_ = Vector2(region->uvs[BLX], region->uvs[BLY]);
vertices[1].uv_ = Vector2(region->uvs[ULX], region->uvs[ULY]);
vertices[2].uv_ = Vector2(region->uvs[URX], region->uvs[URY]);
vertices[3].uv_ = Vector2(region->uvs[BRX], region->uvs[BRY]);
```

I kept the old names to minimize the amount of changes.

Reply:
Needless to say, this only comes into play if there's a spine runtime at **Source/ThirdParty/spine** because all that code is inside a `#ifdef URHO3D_SPINE` ... `#endif` section.

Reply:
I see. In that case it would be great if you can add one or two liner of comment in the code to explain where it is being used. I have no other issues with this PR. Thanks.

Reply:
Done. Thanks!